registry/handlers/app.go

```diff
@@ -577,3 +578,7 @@ func startUploadPurger(storageDriver storagedriver.StorageDriver, log ctxu.Logge
 	}()
 
 }
+
+func (app *App) logError(context context.Context, status int, content interface{}) {
```

Comment:
No need to include `status` in arguments. Let's break out the error into its code, message and detail in the context. I was thinking about doing "err.code", "err.message" and "err.detail".